test-suite/output/Projections.out

```diff
@@ -1,17 +1,20 @@
 fun S : store => S.(store_funcs)
      : store -> host_func
-a =
-fun A : Type =>
-let B := A in fun (C : Type) (u : U A C) => (A, B, C, c _ _ u)
-     : forall A : Type,
-       let B := A in
-       forall C : Type, U A C -> Type * Type * Type * (B * A * C)
+Record U (A : Type) (B : Type := A) (C : Type) : Type := Build_U
+  { c : (A * A * C)%type;  a := (A, A, C, c);  b : a = a } as u.
```

Comment:
It seems the B uses got unfolded (declaration is `{c:B*A*C;a:=(A,B,C,c);b:a=a}`), is it hard to keep them?

Reply:
Interesting! Probably it was already unfolded in the type of `Build_U` in kernel. Will have to be checked.

Reply:
It was indeed (only) a printing bug, so relevant for the PR. Fixed.